Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -7328,6 +7330,15 @@ class BBQJIT {
         return { };
     }
 
+    ALWAYS_INLINE void willParseOpcode()
+    {
+        m_pcToCodeOriginMapBuilder.appendItem(m_jit.label(), CodeOrigin(BytecodeIndex(m_parser->currentOpcodeStartingOffset())));
+    }
+
+    ALWAYS_INLINE void didParseOpcode()
```

Comment:
Do you expect you'll need this too? Seems currently unused in all the generators.

Reply:
I think this is good anyway, for debugging (e.g., dumping some loggings here).